.azure/app-cloud-e2e.yml

```diff
@@ -67,57 +67,42 @@ jobs:
         'App: v0_app':
           name: "v0_app"
           dir: "public"
-          queue_type: "redis"
```

Comment:
@ethanwharris why do we drop redis?

Reply:
Yes, this won't be supported going forward.